libfreerdp/core/gateway/rts.c

```diff
@@ -887,7 +887,7 @@ int rts_recv_flow_control_ack_pdu(rdpRpc* rpc, BYTE* buffer, UINT32 length)
 	BYTE ChannelCookie[16];
 
 	offset = 24;
-	offset += rts_flow_control_ack_command_read(rpc, &buffer[offset], length - offset,
```

Comment:
Well, I don't see why we update variables if we don't use them. Would 
/*offset +=*/ rts_flow_control_ack_command_read(rpc, &buffer[offset], length - offset,
be ok for you @awakecoding ? That would keep the code layout and would please the clang checker...